pkg/services/object/acl/acl_test.go

```diff
@@ -0,0 +1,47 @@
+package acl
+
+import (
+	"testing"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/acl"
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
+	"github.com/nspcc-dev/neofs-api-go/v2/session"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOriginalTokens(t *testing.T) {
+	sTokenBody := new(session.SessionTokenBody)
+	sTokenBody.SetID([]byte("ID"))
+
+	sToken := new(session.SessionToken)
+	sToken.SetBody(sTokenBody)
+
+	owner := new(refs.OwnerID)
+	owner.SetValue([]byte("Owner ID"))
+
+	bTokenBody := new(acl.BearerTokenBody)
+	bTokenBody.SetOwnerID(owner)
+
+	bToken := new(acl.BearerToken)
```

Comment:
Now we can just use `sessiontest.GenerateBearerToken` and `acltest.GenerateBearerToken`